modules/lazyload/assets/js/frontend.js

```diff
@@ -0,0 +1,34 @@
+window.addEventListener( 'DOMContentLoaded', () => {
+    const dataAttribute = 'data-e-bg-src';
+    const lazyloadBackgrounds = document.querySelectorAll( `[${ dataAttribute }]:not(.lazyloaded)` );
+    const customeSelectors = [
+        {
+            elementType: 'column',
+            backgroundSelector: 'div:nth-of-type(1)',
+        },
+        {
+            elementType: 'widget',
+            backgroundSelector: '.elementor-widget-container',
+        },
+    ];
```

Comment:
Move to a data attribute (e.g. `data-e-lazyload-selector``) in order to allow 3rd party elements to define their custom selector.

Reply:
done